src/Cache/CacheEngine.php

```diff
@@ -198,14 +198,14 @@ public function deleteMultiple($keys): bool
     {
         $this->ensureValidType($keys);
 
+        $result = true;
         foreach ($keys as $key) {
-            $result = $this->delete($key);
-            if ($result === false) {
-                return false;
+            if (!$this->delete($key)) {
+                $result = false;
             }
         }
 
-        return true;
+        return $result;
```

Comment:
We should update the docblock to explain it attempts to delete all even if there was an error.